packages/trpc/server/routers/viewer.tsx

```diff
@@ -1192,6 +1192,24 @@ const loggedInViewerRouter = router({
     )
     .mutation(async ({ ctx, input }) => {
       const currentMetadata = userMetadata.parse(ctx.user.metadata);
+      const credentials = ctx.user.credentials;
+      const foundApp = getApps(credentials).filter((app) => app.slug === input.appSlug)[0];
+
+      if (!foundApp) throw new TRPCError({ code: "BAD_REQUEST", message: "App not found" });
```

Comment:
Added a bit more validation so you cant break things

Reply:
Given the condition, I think the error message should probable be "App not installed"

Reply:
```suggestion
      if (!foundApp) throw new TRPCError({ code: "BAD_REQUEST", message: "App not installed" });
```